flex/02-flex-header/style.css

```
@@ -1,6 +1,10 @@
 .header  {
   font-family: monospace;
   background: papayawhip;
```

Please always write in the hash value.
```suggestion
  background: #FFEFD5;
```